dev-support/docker/Dockerfile_debian_10

```diff
@@ -54,6 +49,20 @@ RUN apt-get -q update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+#####
+# For installing the latest packages
+#####
+RUN echo 'deb http://deb.debian.org/debian testing main' >> /etc/apt/sources.list
```

Comment:
If you set this up, why does the packages.json approach do not work?
I would prefer to install using the json approach than going back to coding for each.

Reply:
> If you set this up, why does the packages.json approach do not work?

Ok. Let me check if we can do that.

Reply:
@goiri If I set this at the beginning, everytime the CI creates the Debian 10 build environment, it pulls the latest packages from Debian testing. This is broken currently - i.e. when it tries to install python3, it fails in installing a dependency -

```
[2021-09-08T00:21:22.744Z] #13 11.28 Preparing to unpack .../libc6_2.31-17_amd64.deb ...
[2021-09-08T00:21:23.260Z] #13 11.46 Checking for services that may need to be restarted...
[2021-09-08T00:21:23.260Z] #13 11.48 Checking init scripts...
[2021-09-08T00:21:23.260Z] #13 11.50 Unpacking libc6:amd64 (2.31-17) over (2.28-10) ...
[2021-09-08T00:21:26.290Z] #13 14.38 Setting up libc6:amd64 (2.31-17) ...
[2021-09-08T00:21:26.290Z] #13 14.42 /usr/bin/perl: error while loading shared libraries: libcrypt.so.1: cannot open shared object file: No such file or directory
[2021-09-08T00:21:26.290Z] #13 14.42 dpkg: error processing package libc6:amd64 (--configure):
[2021-09-08T00:21:26.290Z] #13 14.42  installed libc6:amd64 package post-installation script subprocess returned error exit status 127
[2021-09-08T00:21:26.291Z] #13 14.43 Errors were encountered while processing:
[2021-09-08T00:21:26.291Z] #13 14.43  libc6:amd64
[2021-09-08T00:21:26.291Z] #13 14.46 E: Sub-process /usr/bin/dpkg returned an error code (1)
[2021-09-08T00:21:27.867Z] #13 ERROR: executor failed running [/bin/bash -o pipefail -c apt-get -q update     && apt-get -q install -y --no-install-recommends python3     && apt-get -q install -y --no-install-recommends $(pkg-resolver/resolve.py debian:10)     && apt-get clean     && rm -rf /var/lib/apt/lists/*]: exit code: 100
```

Hence, I'm adding the Debian testing as the source only for installing GCC 10. Let me see if I can pin the Debian testing packages to an older date so that we don't have to deal with this.

Reply:
@goiri I was able to avoid this approach by using bullseye instead, ultimately taking the packages.json approach only.